tests/test_invariant_wf_cliprdr.c

```
@@ -0,0 +1,57 @@
+#include <check.h>
+#include <stdlib.h>
+
+#include "../libs/clipboard/src/windows/wf_cliprdr.c"
+
+static BOOL wf_cliprdr_stream_count_within_limit(ULONG stream_count)
+{
+	return stream_count <= WF_CLIPRDR_MAX_STREAMS;
+}
+
+START_TEST(test_stream_count_within_limit_accepts_boundary_values)
```

Cppcheck (2.20.0)

[style] The function 'START_TEST' should have static linkage since it is not used outside of its translation unit.

(staticFunction)